utils/test/src/main/kotlin/ProjectConfig.kt

```diff
@@ -34,7 +34,7 @@ class ProjectConfig : AbstractProjectConfig() {
         OrtProxySelector.install()
     }
 
-    override fun extensions() =
+    override val extensions =
```

Comment:
I guess we also [need to](https://kotest.io/docs/release6/#project-configuration-location) either make this file use the `io.kotest.provided` package, or set the `` property `kotest.framework.config.fqn` for it be picked up correctly.

Reply:
But we can [remove these](https://kotest.io/docs/release6/#removed-classpath-scanning) now:

https://github.com/oss-review-toolkit/ort/blob/94189788916f578a04ce45fc03c45cb3dd656565/utils/test/src/main/resources/kotest.properties#L18-L19

Reply:
Sorry,

https://github.com/oss-review-toolkit/ort/blob/94189788916f578a04ce45fc03c45cb3dd656565/utils/test/src/main/resources/kotest.properties#L21

can be removed as well, see

https://github.com/kotest/kotest/commit/bce04b119a4c1e5f50ff08f90c617497e6f2e48e#diff-2ccbda8b5717a8d89021eff32183888b555d5054ba2e16c199ad954fcad9371fL131